### PR TITLE
Upgrade to version 3.0.1

### LINKED ISSUE #191: Upgrade testing branch to 3.0.0-rc.1
New version is pre-released here : https://github.com/Chocobozzz/PeerTube/releases/tag/v3.0.0-rc.1

May upgrade testing branch like https://github.com/groblochon/peertube_ynh, it's working for me with live stream feature (install only).

We need to open 1935 port :

    $ yunohost firewall allow TCP 1935

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Level 6*
- *New version upstream*
- *Fix #191*

## Solution
- *Upgrade to version 3.0.1*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---

### CODE REVIEW

**Review by ericgaspar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Servers are run independently by different people and organizations. They can ap
 
 By watching a video, you help the hosting provider to broadcast it by becoming a broadcaster of the video yourself. Each instance doesn’t need much money to broadcast the videos of its users.
 
-**Shipped version:** 2.4.0
+**Shipped version:** 3.0.0
```
Comment (arthurlogilab) [inline README.md:30]:
```suggestion
**Shipped version:** 3.0.1
```

File: README_fr.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Les serveurs sont gérés indépendamment par différentes personnes et organisa
 
 En regardant une vidéo, vous aidez l'hébergeur à la diffuser en devenant vous-même un diffuseur de la vidéo. Chaque instance n'a pas besoin de beaucoup d'argent pour diffuser les vidéos de ses utilisateurs.
 
-**Version incluse :** 2.4.0
+**Version incluse :** 3.0.0
```
Comment (arthurlogilab) [inline README_fr.md:26]:
```suggestion
**Version incluse :** 3.0.1
```

File: conf/app.src [hunk 3]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/Chocobozzz/PeerTube/releases/download/v2.4.0/peertube-v2.4.0.tar.xz
-SOURCE_SUM=3e680fff23584608bae464731bcf08cb2126f4e07f42a1547a3812674da3b315
+SOURCE_URL=https://github.com/Chocobozzz/PeerTube/releases/download/v3.0.0/peertube-v3.0.0.tar.xz
```
Comment (arthurlogilab) [inline conf/app.src:1]:
```suggestion
SOURCE_URL=https://github.com/Chocobozzz/PeerTube/releases/download/v3.0.1/peertube-v3.0.1.tar.xz
```

File: conf/app.src [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+SOURCE_SUM=a586cec1e54823b673b8bc29cd44c8df0f100f682afc257172da1ec7e76699c9
```
Comment (arthurlogilab) [inline conf/app.src:2]:
```suggestion
SOURCE_SUM=85d2a99d3bc30eeafc9201bb4f760e37c0132500ce03909669157e66f813f8e9
```

File: manifest.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
         "en": "Video streaming platform using P2P directly in the web browser, connected to a federated network",
         "fr": "Plateforme de diffusion vidéo par P2P directement dans le navigateur, et connectée à un réseau fédéralisé"
     },
-    "version": "2.4.0~ynh1",
+    "version": "3.0.0~ynh1",
```
Comment (arthurlogilab) [inline manifest.json:9]:
```suggestion
    "version": "3.0.1~ynh1",
```

**Review by arthurlogilab — APPROVED** [review]

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Pull request merged**